config/serverless.yml

```diff
@@ -77,3 +77,6 @@ console.autocompleteDefinitions.endpointsAvailability: serverless
 
 # Allow authentication via the Elasticsearch JWT realm with the `shared_secret` client authentication type.
 elasticsearch.requestHeadersWhitelist: ["authorization", "es-client-authentication"]
+
+# FOR LOCAL TESTING - AWAITING MERGE OF https://github.com/elastic/kibana/pull/162149
+server.restrictInternalApis: true
```

Comment:
This will be present once https://github.com/elastic/kibana/pull/162149 is merged to main.

Reply:
And reverted .... https://github.com/elastic/kibana/pull/163296/files